src/components/record/RecordForm.js

```diff
@@ -125,11 +125,11 @@ export default class RecordForm extends Component {
   }
 
   get allowEditing() {
-    const {session, collection, record} = this.props;
+    const {session, bucket, collection, record} = this.props;
     if (record) {
-      return canEditRecord(session, record);
+      return canEditRecord(session, bucket, collection, record);
     } else {
-      return canCreateRecord(session, collection);
+      return canCreateRecord(session, bucket, collection);
```

Comment:
We may want to check for a record being provided in the `canCreateRecord` function instead.